specification/storagecache/resource-manager/Microsoft.StorageCache/stable/2021-05-01/storagecache.json

```diff
@@ -1628,7 +1632,11 @@
         },
         "rotationToLatestKeyVersionEnabled": {
           "type": "boolean",
-          "description": "Specifies whether the service will automatically rotate to the newest version of the key in the Key Vault."
+          "description": "Specifies whether the service will automatically rotate to the newest version of the key in the Key Vault.",
```

Comment:
Confirmation of this behavior is still pending. Please don't merge until the _expected_ behavior is confirmed/finalized.

Reply:
Internal Bug: 10014874

Reply:
@rebecca337 Looks like this needs to be mutable per latest PM guidance last Friday (5/28). Please remove the "create-only" tags.

Reply:
Removed the tag!